jmh-benchmarks/README.md

```diff
@@ -0,0 +1,63 @@
+###JMH-Benchmark module
```

Comment:
OK to leave it like it is for now. We can rename it later if we decide it's better.